astropy/units/format/vounit.py

```diff
@@ -9,6 +9,7 @@
 import warnings
 from typing import TYPE_CHECKING
 
+from astropy.units.errors import UnitScaleError, UnitsWarning
```

Comment:
Import thingy again. Here, `core` is probably used too much to remove it (which arguably argues for not doing so elsewhere too, though I see below that `quantity` does just import a lot from it).